core-container/entry.sh

```diff
@@ -48,9 +48,9 @@ function get_kcp_kubeconfig() {
 
 function create_kcp_provider_object() {
     get_kcp_kubeconfig     # kcp is created in a seperate container 
-    kubectl --kubeconfig $SPACE_MANAGER_KUBECONFIG delete secret -n ${NAMESPACE} kcpsec > /dev/null 2>&1 || true
-    kubectl --kubeconfig $SPACE_MANAGER_KUBECONFIG create secret generic -n ${NAMESPACE} kcpsec --from-file=kubeconfig=$kcp_kubeconfig --from-file=external=$external_kcp_kubeconfig
-    kubectl --kubeconfig $SPACE_MANAGER_KUBECONFIG apply -f - <<EOF
+    kubectl --kubeconfig $SM_CONFIG delete secret -n ${NAMESPACE} kcpsec > /dev/null 2>&1 || true
```

Comment:
This deletion is a concurrency bug. If there is a Secret object to be deleted then that means that a concurrent process might be trying to read it.